x/treasury/internal/types/util.go

```diff
@@ -45,9 +46,9 @@ func NewMsgOracleBurn(fromDid did.Did, amount sdk.Coins,
 	}
 }
 
-func CheckNotEmpty(value string, name string) (valid bool, err sdk.Error) {
+func CheckNotEmpty(value string, name string) (valid bool, err error) {
 	if strings.TrimSpace(value) == "" {
-		return false, sdk.ErrUnknownRequest(name + " is empty.")
+		return false, sdkerrors.Wrap(sdkerrors.ErrUnknownRequest, "name is empty")
```

Comment:
Please include the variable name in the message, rather than literally the word "name"

Reply:
changed